app/src/main/res/values/strings.xml

```diff
@@ -104,4 +104,14 @@
     <string name="are_you_sure_you_want_to_cancel">Are you sure you want to cancel?</string>
     <string name="edit">edit</string>
     <string name="avatar_shop">avatar_Shop</string>
+    <string name="two_dot">:</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.two_dot` appears to be unused
 Unused resources make applications larger and slow down builds.